core/src/main/java/nl/nn/adapterframework/processors/CorePipeLineProcessor.java

```diff
@@ -186,58 +186,62 @@ public PipeLineResult processPipeLine(PipeLine pipeLine, String messageId, Messa
 
 				if (forwardTarget instanceof PipeLineExit) {
 					PipeLineExit plExit= (PipeLineExit)forwardTarget;
-					boolean outputWrapError = false;
-					IPipe outputWrapper = pipeLine.getOutputWrapper();
-					if (outputWrapper !=null) {
-						log.debug("wrapping PipeLineResult");
-						PipeRunResult wrapResult = pipeProcessor.processPipe(pipeLine, outputWrapper, message, pipeLineSession);
-						if (wrapResult!=null && !wrapResult.getPipeForward().getName().equals("success")) {
-							PipeForward wrapForward=wrapResult.getPipeForward();
-							if (wrapForward.getPath()==null) {
-								throw new PipeRunException(pipeToRun,"forward ["+wrapForward.getName()+"] of outputWrapper has emtpy forward path");
-							}
-							log.warn("setting next pipe to ["+wrapForward.getPath()+"] due to wrap fault");
-							outputWrapError = true;
-							pipeToRun = pipeLine.getPipe(wrapForward.getPath());
-							if (pipeToRun==null) {
-								throw new PipeRunException(pipeToRun,"forward ["+wrapForward.getName()+"], path ["+wrapForward.getPath()+"] does not correspond to a pipe");
+					if(!plExit.getEmptyResult()) {
```

Comment:
I don't know why it looks i changed a lot. I just added this check and corresponding else block.